wallet/src/Pos/Util/Mnemonic.hs

```diff
@@ -26,6 +26,9 @@ module Pos.Util.Mnemonic
        , mnemonicToAesKey
        , entropyToMnemonic
        , entropyToByteString
+
+       -- * Reexports
+       , ValidMnemonicSentence
```

Comment:
See comment above, I don't think it's needed here. I believe that we do want to treat `Mnemonic 9` and `Mnemonic 12` as separated types throughout the codebase, because they have different semantics in the end.